corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js

```diff
@@ -171,6 +171,163 @@ hqDefine("hqwebapp/js/knockout_bindings.ko", ['jquery', 'knockout', 'jquery-ui/u
         },
     };
 
+    ko.bindingHandlers.multi_sortable = {
+        updateSortableList: function(itemList) {
+            _(itemList()).each(function(item, index) {
+                if (item._sortableOrder === undefined) {
+                    item._sortableOrder = ko.observable(index);
+                } else {
+                    item._sortableOrder(index);
+                }
+            });
+        },
+        getList: function(valueAccessor) {
+            /* this function's logic follows that of ko.bindingHandlers.foreach.makeTemplateValueAccessor */
```

Comment:
So this comment gives me a clue to look up the source for makeTemplateValueAccessor, but I'd argue that instead of matching the code closely it'd be better to make this code slightly more readable - use _.isArray instead of `typeof unwrappedValue.length === "number"`, possibly copy over the comments from makeTemplateValueAccessor. There's a good chance this code will get out of "sync" with makeTemplateValueAccessor, might as well make it readable.

Reply:
https://github.com/dimagi/commcare-hq/pull/20895/commits/5114056217c5b8baf2f995e0edc978a8a0f1a1a4